aptoide-database/build.gradle

```diff
@@ -55,4 +63,12 @@ dependencies {
   implementation "nl.littlerobots.rxlint:rxlint:${Rx_lint_version}"
   // And ProGuard rules for RxJava!
   implementation "com.artemzin.rxjava:proguard-rules:${RXJAVA_PROGUARD_RULES_VERSION}"
+  implementation 'androidx.room:room-runtime:2.2.3'
+  annotationProcessor 'androidx.room:room-compiler:2.2.3'
+
+  // RxJava support for Room
+  implementation "androidx.room:room-rxjava2:${ROOM_VERSION}"
+
+  implementation "io.reactivex.rxjava2:rxjava:2.2.10"
+  implementation "io.reactivex.rxjava2:rxandroid:2.1.1"
```

Comment:
Any reason not to use project level variables here?

Reply:
already changed it in another branch